cli/src/query.rs

```diff
@@ -0,0 +1,49 @@
+use super::error::{Error, Result};
+use std::fs;
+use std::io::{self, Write};
+use std::path::Path;
+use tree_sitter::{Language, Parser, Query, QueryCursor};
+
+pub fn query_files_at_paths(
+    language: Language,
+    paths: Vec<&Path>,
+    query_path: &Path,
+) -> Result<()> {
```

Comment:
This file implements a new `tree-sitter query` command. This command isn't really very useful right now, but it was useful at a certain stage for testing queries from the command line.

Right now, it just prints a bunch of messages to stdout. I might change it in the future to just print the contents of the file, but with query matches colored in some way. Kind of like the `tree-sitter highlight` command, but where you'd specify an arbitrary query filename, instead of always looking for the *syntax-highlighting* query.